src/tree_sitter.rs

```diff
@@ -97,7 +97,7 @@ pub fn apply_query(
     Ok(atoms)
 }
 
-fn capture_name<'a, 'b>(query: &'a Query, capture: &'b QueryCapture) -> &'a str {
+fn capture_name<'a>(query: &'a Query, capture: &QueryCapture) -> &'a str {
```

Comment:
Is this equivalent to:

```rust
fn capture_name<'a>(query: &'a Query, capture: &'_ QueryCapture) -> &'a str
```

i.e., The lifetime of `capture` doesn't matter because the compiler can prove it's within `'a`?

Reply:
Not necessarily within `'a`, but yes, it doesn't matter. See https://doc.rust-lang.org/book/ch10-03-lifetime-syntax.html#lifetime-elision